.gitignore

```diff
@@ -59,3 +59,5 @@ typings/
 
 # next.js build output
 .next
+
+components/Users.js
```

Comment:
why is this component ignored?

Reply:
so after talking I know this is because you are saving the secrets there, but instead you should use env files for that